sdk/eventgrid/eventgrid/review/eventgrid.api.md

```diff
@@ -1172,13 +1194,14 @@ export interface SystemEventNameToEventData {
     "Microsoft.Communication.ChatMessageDeleted": AcsChatMessageDeletedEventData;
     "Microsoft.Communication.ChatMessageEdited": AcsChatMessageEditedEventData;
     "Microsoft.Communication.ChatMessageReceived": AcsChatMessageReceivedEventData;
-    "Microsoft.Communication.ChatParticipantAddedToThread": AcsChatParticipantAddedToThreadEventData;
     "Microsoft.Communication.ChatParticipantAddedToThreadWithUser": AcsChatParticipantAddedToThreadWithUserEventData;
-    "Microsoft.Communication.ChatParticipantRemovedFromThread": AcsChatParticipantRemovedFromThreadEventData;
     "Microsoft.Communication.ChatParticipantRemovedFromThreadWithUser": AcsChatParticipantRemovedFromThreadWithUserEventData;
     "Microsoft.Communication.ChatThreadCreatedWithUser": AcsChatThreadCreatedWithUserEventData;
+    "Microsoft.Communication.ChatThreadParticipantAdded": AcsChatParticipantAddedToThreadEventData;
```

Comment:
Any concerns about the string not matching the model type naming anymore?
We seem to have this convention:
"Acs" + type name from the string + "EventData" should match the model type.

Reply:
I don't really have a concern.  In practice, you will not likely utter these names in source, rather you'll just call `isSystemEvent(...)` and TypeScript will know if it returns true the object you tested is now of that shape and you'll get good IntelliSense.  Note that these names are also a typescript only concept.